source/gwork/source/Anim.cpp

```diff
@@ -113,9 +113,9 @@ static float GetEased(float fTime, float fEase)
 {
     // Ease in and Out if ease is < 0
     if (fEase < 0)
-        return -fTime/2*(std::cosf(3.14159f*fTime)-1);
+        return -fTime/2*(std::cos(3.14159f*fTime)-1);
```

Comment:
C++11 cos is overloaded for float and double, so just using cos is fine; it's not guaranteed C99 cosf will be put under std:: (and it isn't on VS2015, or GCC under linux).

This is one error preventing GWork from compiling on my system